formats/service_manual_service_standard/publisher/details.json

```diff
@@ -5,6 +5,11 @@
   "properties": {
     "body": {
       "$ref": "#/definitions/body"
+    },
+    "poster_url": {
```

Comment:
I think a string is fine for now. We can revisit when updating the standard actually moves into publisher proper, rather than a rake task.